eradiate/scenes/measure.py

```diff
@@ -201,6 +205,7 @@ class TargetRectangle(Target):
         Unit-enabled field (default: cdu[length]).
     """
 
+    # fmt: off
```

Comment:
This will have unintended side effects: you're basically disabling formatting on half of the file. This is okay for editing, but you should not leave the marks when you're done: other maintainers working on the code will likely want to format code and they shouldn't be looking at formatter markup to have it work.

We're not enforcing black yet, it's a very disruptive change (I imagine you've been as scared as I have by how meaningless diffs become when you black-format the entire project); but I haven't changed my mind and I want it to be our formatting tool in the future. A possible solution for transition (haven't tried it yet but seems capable) is [darker](https://pypi.org/project/darker/): it will apply formatting only to regions of the code which have been modified since you've last committed. The tool is flexible and has many options as well as IDE integration instructions. Only minor issue: it's not packaged for Conda (but I can now create a recipe to package it myself).

```suggestion
```